doc/caveats.md

```diff
@@ -28,12 +28,11 @@ cross-platform notification of events in git repositories (e.g. pushes).
 Using such a mechanism, contributors could notify a project maintainer about
 changes in their public repositories, including issues or messages, via a public
 API. A maintainer could set up automated fetches of new issues, importing new
-issues and messages. This is, however unrelated and still work in progress.
+issues and messages. This is however, unrelated and still work in progress.
 Additionally, it would still require reporters of issues to expose them via a
 public repository.
 
 It was also suggested that maintainers or service providers could provide
 special public bug repositories with public push access. Those repositories
 would have specially crafted hooks installed, implementing some sort of access
 control, e.g. preventing issues from being deleted by unauthorized actors.
-
```

Comment:
I like keeping a newline at the end of text and source files. In my experience, it helps with merges in some situations.

Reply:
ouh yes I noticed when opening the other files, then forgot to undo this, sorry!
will change

Reply:
done.